src/main/resources/schemas/sscs_daily_list.json

```diff
@@ -31,34 +31,6 @@
       "description": "Can be an array of objects as such will be a collective and use plurals for object arrays (clerks and scribes etc included)",
       "type": "object",
       "properties": {
-        "johTitle": {
```

Comment:
The SSCS list too does not currently have the isPresiding flag in the schema. Just checking this is expected?

Reply:
Checked with Chris and this one did need the isPresiding flag adding, I've made those changes now